Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -485,7 +485,7 @@ private void InitializeServiceLocator()
             if (ActiveProfile.IsCameraSystemEnabled)
             {
                 DebugUtilities.LogVerbose("Begin registration of the camera system");
-                object[] args = { ActiveProfile.CameraProfile };
+                object[] args = { ActiveProfile.CameraProfile, ActiveProfile.FloorHeight };
```

Comment:
Is there some reason that the floor height is not part of the camera profile?

Reply:
It's not part of the camera profile because it seems like it should belong in the same place as the targetExperienceScale property. In general these seem like "broad" MRTK settings, and I feel like they should be put at the topmost level from an organizational standpoint, though I'm open to other suggestions.